### PR TITLE
python 3 compatibility

### PR DESCRIPTION
**Pull request by stephanerobert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,8 +20,8 @@ implemented on top of redlock to manage access to the requested lock. This queue
 Requirements
 ------------
 
- * python 2.7
- * tox
+ * python 2.7 or python 3.4
+ * redlock-py
```
Comment (mat128) [inline README.md]: I think we should remove this paragraph altogether. Saying we require tox isn't all that useful, given we provide "how to run the tests" below.

File: redlock_fifo/extendable_redlock.py [hunk 2]
```diff
@@ -66,17 +65,22 @@ def stop_autoextend(self, lock):
...
+class AutoExtendableLockThread(Thread):
     def __init__(self, redlock, lock, every_ms, new_ttl):
         super(AutoExtendableLockThread, self).__init__()
         self.lock = lock
         self.redlock = redlock
         self.every_ms = every_ms
         self.new_ttl = new_ttl
+        self.extend = True
+
+    def run(self):
```
Comment (mat128) [inline redlock_fifo/extendable_redlock.py:77]: Good job =)

File: tox.ini [hunk 3]
```diff
@@ -1,7 +1,6 @@
 [tox]
-envlist = py27
+envlist = py{27,34}
```
Comment (mat128) [inline tox.ini:2]: We should also add python 3 test run to .travis.yml

```
python:
  - "2.7"
  - "3.4"

deploy:
  on:
    python: 3.4
```

File: README.md (hunk 1 above)
Comment (stephanerobert) [inline README.md]: I actually removed tox from the requirements and added redlock-py

File: tox.ini (hunk 3 above)
Comment (stephanerobert) [inline tox.ini:2]: Good point, please teach me the way to travis ci :-)

**Comment by fbouliane** [comment]
Is this modification force python34 usage ?
I don't have python34 and I have this error on `tox`

```
~/git/netman$ tox --notest
GLOB sdist-make: /Users/fbouliane/git/netman/setup.py
py27 inst-nodeps: /Users/fbouliane/git/netman/.tox/dist/redlock-fifo-0.0.1.dev22.zip
py27 installed: You are using pip version 7.1.2, however version 8.0.2 is available.,You should consider upgrading via the 'pip install --upgrade pip' command.,fakeredis==0.6.2,mock==1.0.1,nose==1.3.7,PyHamcrest==1.8.5,redis==2.10.5,redlock-fifo==0.0.1.dev22,redlock-py==1.0.6,six==1.10.0,wheel==0.24.0
py34 create: /Users/fbouliane/git/netman/.tox/py34
ERROR: InterpreterNotFound: python3.4
______________________________________________________________________________________________________________________________ summary ______________________________________________________________________________________________________________________________
  py27: skipped tests
ERROR:   py34: InterpreterNotFound: python3.4
```

**Comment by stephanerobert** [comment]
yep, you'll need to install python 3.4 to run tox

File: .gitignore [hunk 4]
```diff
@@ -1,8 +1,8 @@
+.idea
```
Comment (fbouliane) [inline .gitignore:1]: This shouldn't be in your global .gitignore ?

**Comment by fbouliane** [comment]
:+1:

**Comment by JoProvost** [comment]
:+1:

**Pull request merged**